test/Microsoft.Azure.Devices.Edge.Test.Common/windows/EdgeDaemon.cs

```diff
@@ -64,14 +64,20 @@ public async Task InstallAsync(Option<string> packagesPath, Option<Uri> proxy, C
 
         public async Task ConfigureAsync(Func<DaemonConfiguration, Task<(string, object[])>> config, CancellationToken token, bool restart)
         {
-            string configYamlPath =
-                Environment.GetFolderPath(Environment.SpecialFolder.CommonApplicationData) + @"\iotedge\config.yaml";
+            // These tests do not run on Windows; use empty file paths.
```

Comment:
Better to wipe the Windows and Kubernetes E2E tests than update them with stubs. That is what I have been doing throughout the code.

Reply:
Removed Windows tests.